.gitignore

```diff
@@ -26,6 +26,7 @@ _testmain.go
 # Tool related configurations
 .ackrc
 .envrc
+.idea
```

Comment:
This should go in your `~/.gitignore` instead of in this project. The above files probably shouldn't even be in here.